VKApp/VKApp/TabBar/First/SortedFriendsPhotosViewController.swift

```diff
@@ -50,7 +50,8 @@ final class SortedFriendsPhotosViewController: UIViewController {
 
     private func fetchImages() {
         usersPhotoURLPath.forEach { [weak self] url in
```

Comment:
```suggestion
        let networkService = NetworkService()
        usersPhotoURLPath.forEach { [weak self] url in
```
 создаем один раз перед циклом